frontend/src/components/JoinUs.vue

```diff
@@ -13,10 +13,18 @@
       <v-col>
         <v-card flat dark tile color="transparent">
           <p class="content">
-            Kodujesz? Jesteś grafikiem? UX-owcem? Robisz cokolwiek innego, co
-            może się przydać w kodowaniu dla Poznania? Dołącz do nas! Spotykamy
-            się w każdą środę od godz. 18:00 w firmie STX Next przy ul. Mostowej
-            38 niedaleko Nowej Gazowni (Pawilonu) w parku Stare Koryto Warty.
+            Kodujesz? Testujesz? Zajmujesz się grafiką albo UXem? A może
+            sprawnie zarządzasz projektami IT?
+          </p>
+          <br />
+          <p class="content">
+            W czasie pandemii spotykamy się online, co tydzień w środę o 18:00.
```

Comment:
It looks better: `W czasie pandemii spotykamy się zdalnie co tydzień w środę o 18:00.`